src/DIRAC/ConfigurationSystem/Agent/RucioSynchronizerAgent.py

```diff
@@ -336,7 +336,7 @@ def execute(self):
                 if account not in listAccounts:
                     self.log.info("Will create %s with associated DN %s", account, dn)
                     try:
-                        client.add_account(account=account, type="USER", email=email)
+                        client.add_account(account=account, type_="USER", email=email)
```

Comment:
I thought you agreed (https://github.com/DIRACGrid/DIRAC/issues/5662#issuecomment-982815133) that it's best to do this treating them as positional arguments rather than keyword arguments?

Reply:
we should use the keyword if and only if they are defined as keyword arguments.
It's not the case here
https://github.com/rucio/rucio/blob/4d9d934e976114d54d37d75b8bb7c710ae14188b/lib/rucio/api/account.py#L39

Reply:
Yes, but this is the link. The method is here:
[https://github.com/rucio/rucio/blob/4d9d934e976114d54d37d75b8bb7c710ae14188b/lib/rucio/client/accountclient.py#L49](https://github.com/rucio/rucio/blob/4d9d934e976114d54d37d75b8bb7c710ae14188b/lib/rucio/client/accountclient.py#L49)